<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>561</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>上传工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="version_label">
     <property name="text">
      <string>version</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>流程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPipeline">
       <item>
        <property name="text">
         <string>灯光</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>渲染</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>合成</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>输出</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QToolButton" name="toolButtonOpenDir">
       <property name="text">
        <string>输入目录</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dirEdit">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="dirButton">
       <property name="text">
        <string>浏览</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QListView" name="listView">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="labelCount">
         <property name="text">
          <string>local/total</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxBurnIn">
         <property name="toolTip">
          <string>双击时使用burn-in版本(如果有)</string>
         </property>
         <property name="text">
          <string>辅助信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonRefresh">
         <property name="text">
          <string>刷新</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonSelectAll">
         <property name="text">
          <string>全选</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonReverseSelection">
         <property name="text">
          <string>反选</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBoxSubmit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>上传时同时进行一次对应镜头的CGTeamWork提交</string>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>备注</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNote">
       <property name="placeholderText">
        <string>default_note_text</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="syncButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>上传</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStatusBar" name="statusBar"/>
   </item>
  </layout>
  <action name="actionSync">
   <property name="text">
    <string>sync</string>
   </property>
  </action>
  <action name="actionDir">
   <property name="text">
    <string>dir</string>
   </property>
  </action>
  <action name="actionOpenDir">
   <property name="text">
    <string>openDir</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>selectAll</string>
   </property>
  </action>
  <action name="actionReverseSelection">
   <property name="text">
    <string>reverseSelection</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>reset</string>
   </property>
  </action>
  <zorder>statusBar</zorder>
  <zorder>version_label</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>syncButton</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dirButton</sender>
   <signal>clicked()</signal>
   <receiver>actionDir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonOpenDir</sender>
   <signal>clicked()</signal>
   <receiver>actionOpenDir</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncButton</sender>
   <signal>clicked()</signal>
   <receiver>actionSync</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>533</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSelectAll</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectAll</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonReverseSelection</sender>
   <signal>clicked()</signal>
   <receiver>actionReverseSelection</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxSubmit</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditNote</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonRefresh</sender>
   <signal>clicked()</signal>
   <receiver>actionReset</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
